Tella/Scenes/Uwazi/Utils/UwaziEntityParser.swift

```diff
@@ -8,77 +8,261 @@
+                }
+                
+                let selectValues = relationshipValues.compactMap({SelectValues(id: $0.id, label: $0.label)})
+
+                prompt = UwaziRelationshipEntryPrompt(id: $0.id ?? "",
+                                                      formIndex: $0.id,
+                                                      type: $0.type ?? "",
+                                                      question: $0.translatedLabel ?? "",
+                                                      content: $0.content ?? "",
+                                                      required: $0.propertyRequired,
+                                                      helpText: $0.translatedLabel,
+                                                      selectValues: selectValues ,
+                                                      name: $0.name)
+                
+            default:
+                prompt = UwaziTextEntryPrompt(id: $0.id ?? "",
+                                              formIndex: $0.id,
+                                              type: $0.type ?? "",
+                                              question: $0.translatedLabel ?? "",
+                                              content: $0.content ?? "",
+                                              required: $0.propertyRequired,
+                                              helpText: $0.translatedLabel,
+                                              selectValues: [],
+                                              name: $0.name)
+            }
+            
+            entryPrompts.append(prompt)
+        }
+    }
+    
+    func updateRelationships(relationships: [UwaziRelationshipList]?) {
+         
+        guard let entryPrompts = entryPrompts.filter({$0.type == .dataRelationship}) as? [UwaziRelationshipEntryPrompt] else {return}
+        
+        _ = entryPrompts.compactMap({ prompt in
+            let values = relationships?.first (where: {$0.id == prompt.content } )?.values ?? []
+            return values.compactMap({SelectValues(id: $0.id, label: $0.label)})
+        })
+    }
+    
+    func saveAnswersToEntityInstance(status:EntityStatus) {
+        
+        var metadata: [String: Any] = [:]
+        var uwaziEntityInstanceFile : [UwaziEntityInstanceFile] = []
+        let entityInstance = self.entityInstance == nil ? UwaziEntityInstance() : self.entityInstance
+        
+        for entryPrompt in entryPrompts {
+            
+            guard let propertyName = entryPrompt.name else { break }
+            
+            switch entryPrompt.type {
+                
+            case .dataTypeText where propertyName == UwaziEntityMetadataKeys.title:
+                guard let entryPrompt = entryPrompt as? UwaziTextEntryPrompt else { continue }
+                entityInstance?.title = entryPrompt.value
+                
+            case .dataTypeText, .dataTypeNumeric, .dataTypeMarkdown,.dataTypeDate:
```

Comment:
for .dataTypeDate this will break because it needs to be parsed as int, otherwise, the Uwazi server will not receive it properly